
int16 pts_map
Pose3D[] keyframes_pose
int16[] keyframes_ID
int16[] n_pts_keyframe
int16[] n_mapped_pts_keyframe
float32 BA_times
int16 BA_num_iter
